Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Data.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents the control that redistributes space between columns or rows of a Grid control.
+    /// </summary>
+    public partial class GridSplitter
+    {
+        // GridSplitter has special Behavior when columns are fixed
+        // If the left column is fixed, splitter will only resize that column
+        // Else if the right column is fixed, splitter will only resize the right column
+        private enum SplitBehavior
+        {
+            Split, // Both columns/rows are star lengths
+            Resize1, // resize 1 only
+            Resize2, // resize 2 only
+        }
+
+        // Only store resize data if we are resizing
+        private class ResizeData
```

Comment:
This looks an awful lot like the code from reference source here:
http://referencesource.microsoft.com/#PresentationFramework/src/Framework/System/Windows/Controls/GridSplitter.cs,844b7b95bff72024

I think this is a violation of the license agreement you signed with this PR and the license agreement of the reference source (but not entirely sure - will have to check with the experts).